src/OptionHandler.cpp

```diff
@@ -110,6 +111,9 @@ static std::unique_ptr<AudioFileReader> createAudioFileReader(
     else if (input_format == FileFormat::Mp3) {
         reader.reset(new Mp3AudioFileReader);
     }
+    else if (input_format == FileFormat::Raw) {
+        reader.reset(new RawAudioFileReader);
```

Comment:
Instead of a new class, I think we can use `SndFileAudioFileReader`, but add a `configure` method that takes in the sample rate and data format, which we'd get from the `Options` object. Something like:

```cpp
reader->configure(
    options.getRawAudioSampleRate(),
    options.getRawAudioBitsPerSample(),
    options.getRawAudioSampleFormat()
);
```

Reply:
Should be done like this now.